test/metric_value_benchmark/statsd_test.go

```diff
@@ -71,58 +130,44 @@ func (t *StatsdTestRunner) validateStatsdMetric(metricName string) status.TestRe
 			Value: dimension.ExpectedDimensionValue{Value: aws.String("value")},
 		},
 	}
-	switch metricName {
-	case "statsd_counter_1":
-		instructions = append(instructions, dimension.Instruction{
-			// CWA adds this metric_type dimension.
-			Key:   "metric_type",
-			Value: dimension.ExpectedDimensionValue{Value: aws.String("counter")},
-		})
-	case "statsd_gauge_1":
-		instructions = append(instructions, dimension.Instruction{
-			// CWA adds this metric_type dimension.
-			Key:   "metric_type",
-			Value: dimension.ExpectedDimensionValue{Value: aws.String("gauge")},
-		})
-	}
-
+	split := strings.Split(metricName, "_")
+	metricType := split[1]
```

Comment:
nit - might as well validate that we have at least 2 values here. don't want to get tripped up by a random out of bounds error

Reply:
I felt it was unnecessary because the `metricName` comes from the global list - `metricNames` which all follow a specific format.

Also I think the test crashing / panicking is just another way to inform the user the test failed.

I will add the check anyways though.